### DISCUSSION TITLE
Why my account can not access by public?

### DISCUSSION THREAD

**Discussion by tzfun** [community/community#102582] [Other Feature Feedback, Questions, & Ideas] [answered]
I have never deleted the account and these projects, and I usually commit the code normally.

I tried several times and found that I can only access my homepage and project after logging in to github, so that others cannot access my project. This will have an impact on my project.

![image](https://github.com/community/community/assets/34815354/8505614e-c0d6-44fc-b84f-c1817c854b05)

Recently active projects:

* https://github.com/tzfun/vue-web-terminal
* https://github.com/tzfun/jvmm
* https://github.com/tzfun/etcd-workbench

This is what I see when I visit my homepage without logging in to my github account.
![image](https://github.com/community/community/assets/34815354/cd9fbaaa-a888-441a-958d-3329c01607c9)

This is what I see when I log in to my github account and visit my homepage.
<img width="1792" alt="image" src="https://github.com/community/community/assets/34815354/eb30c3c0-eb22-451c-bb26-1945dc1e42bc">

I guess the system has restricted my account. Being publicly accessible, I have not done anything illegal.  What should I do next?

**Comment by Cauatn** [comment, marked as answer]
I checked all the links in a separate browser in anonymous mode, and it seems that everything is working fine. 
I think is just a configuration thing on the browser or a anoing incovinient

**Reply by tzfun** [reply]
After that, it was inexplicably better again. I didn't do anything, but this phenomenon did exist during that period. I tried many browsers and I also had many users with the same problem, but it's no longer a problem now. Thank you.

**Reply by Cauatn** [reply]
You're welcome,
I suggest marking one of the answers as 'answered' just to ensure you don't receive any annoying notifications in the future

**Comment by PierreJanineh** [comment]
Your homepage and other repos your provided links for are visible to me. You can be calm.

**Reply by tzfun** [reply]
It seems that there is no problem now, but it was indeed a problem at the time, but that is not important anymore, thank you.

**Comment by paulospx** [comment]
The same here. 
Already opened a ticket with Github it is taking weeks and they haven't yet looked at it.
Getting disappointed with GitHub Support. 
That is something they need to improve.

[a comment by an automated account is not shown]

**Comment by AkashMs24** [comment]
Hello GitHub Support Team,

My GitHub profile is not accessible to others. When anyone visits my profile link:
https://github.com/AkashMs24
they receive a “404 Not Found” error.

However, I am able to log in and see my account normally. I have not deleted my account, and I haven't violated any policy knowingly. This issue is preventing me from sharing my profile for learning and career purposes.

Could you please check if my account has been flagged or restricted and help me restore public visibility?

Thank you for your help.
Username: AkashMs24

**Comment by mikesdatawork** [comment]
I'm having the same experience today.  https://github.com/mikesdatawork 
Same issue.  Same response.  
Going to the main page of the Git I receive a “404 Not Found” error.
